javascript/src/db.js

```diff
@@ -0,0 +1,40 @@
+const cassandra = require('cassandra-driver');
+
+const KEYSPACE = 'carepet';
+
+async function getClient(config, keyspace) {
+  const client = new cassandra.Client({
+    contactPoints: config.hosts,
+    authProvider: new cassandra.auth.PlainTextAuthProvider(
+      config.username,
+      config.password
+    ),
+    localDataCenter: 'datacenter1',
+    keyspace,
+  });
+
+  await client.connect();
```

Comment:
It's not necessary but then first connection will be attempted lazily. You would most of the time want your application to fail early rather than lazily when it comes to the database.